packages/components/package.json

```diff
@@ -27,7 +27,7 @@
     "@fortawesome/free-regular-svg-icons": "^5.15.4",
     "@fortawesome/free-solid-svg-icons": "^5.15.4",
     "@fortawesome/react-fontawesome": "^0.1.15",
-    "@terra-ui-packages/core-utils": "^0.2.0",
+    "@terra-ui-packages/core-utils": "^0.3.0",
```

Comment:
It looks like we don't _need_ this bump, but it also isn't going to hurt anything, so 🤷 .